src/riak_pb_kv_codec.erl

```diff
@@ -173,21 +175,24 @@ decode_content_meta(indexes, PbIndexes, _Pb) ->
     Indexes = [decode_pair(E) || E <- PbIndexes],
     [{?MD_INDEX, Indexes}];
 decode_content_meta(deleted, DeletedVal, _Pb) ->
-    [{?MD_DELETED, DeletedVal}].
-
+    [{?MD_DELETED, DeletedVal}];
+decode_content_meta(ttl, TTLVal, _Pb) ->
+    [{?MD_DELETED, TTLVal}].
 
 %% @doc Convert an rpccontent pb message to an erlang {MetaData,Value} tuple
 -spec decode_content(PBContent::tuple()) -> {metadata(), binary()}.
 decode_content(PbC) ->
-    MD =  decode_content_meta(content_type, PbC#rpbcontent.content_type, PbC) ++
-          decode_content_meta(charset, PbC#rpbcontent.charset, PbC) ++
-          decode_content_meta(encoding, PbC#rpbcontent.content_encoding, PbC) ++
-          decode_content_meta(vtag, PbC#rpbcontent.vtag, PbC) ++
-          decode_content_meta(links, PbC#rpbcontent.links, PbC) ++
-          decode_content_meta(last_mod, PbC#rpbcontent.last_mod, PbC) ++
-          decode_content_meta(usermeta, PbC#rpbcontent.usermeta, PbC) ++
-          decode_content_meta(indexes, PbC#rpbcontent.indexes, PbC) ++
-          decode_content_meta(deleted, PbC#rpbcontent.deleted, PbC),
+    MD = decode_content_meta(content_type, PbC#rpbcontent.content_type, PbC) ++
+         decode_content_meta(charset,  PbC#rpbcontent.charset, PbC) ++
```

Comment:
It looks to me like the second argument of `decode_content_meta` was aligned on each line but the first.